<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contract</class>
 <widget class="QDialog" name="Contract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заключение контракта</string>
  </property>
  <widget class="QComboBox" name="clientBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>выберите клиента</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Клиент:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="placeholderText">
    <string>сумма контракта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>К оплате:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>рублей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сведения о карте:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyCard">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>87</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Купить карту</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newClient">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>38</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый клиент</string>
   </property>
  </widget>
  <widget class="QLabel" name="cardInfo">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>94</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Нет сведений о картах</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
